dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryMessagesTest.java

```diff
@@ -85,14 +88,37 @@ public void testTelemetryMessages() throws Exception {
 
     static void testTelemetryMessages(LibertyServer s, Consumer<List<String>> consoleConsumer) throws Exception {
         String line = s.waitForStringInLog("CWWKF0011I", s.getConsoleLogFile());
-        List<String> linesMessagesLog = s.findStringsInLogs("^(?!.*scopeInfo).*\\[.*$", s.getDefaultLogFile());
-        List<String> linesConsoleLog = s.findStringsInLogs(".*scopeInfo.*", s.getConsoleLogFile());
+        RemoteFile messageLog = s.getDefaultLogFile();
+        RemoteFile consoleLog = s.getConsoleLogFile();
+        s.setMarkToEndOfLog(messageLog);
+        s.setMarkToEndOfLog(consoleLog);
```

Comment:
Since, we are using synchronousWrite, the Console log would get the bridged log file, before message log. We should get the console remote file and set the mark for the console log, before we do the message log.